### PR TITLE
Remove grpc

### PR DESCRIPTION
**Pull request by dturanski**
* Remove grpc related code
* Implement HTTP streaming
* Remove docker, old riff k8s .yaml files
* Travis build only runs tests ATM.

### CODE REVIEW

**Comment by dturanski** [comment]
@matthewmcnew - I just rewrote this to support HTTP streaming, replacing `bottle` with `gevent` to run the function invoker async and use gevent Queues to pass data.  Review and comments appreciated (I apparently don't have perms to assign you as a reviewer).

File: invoker/http_server.py [hunk 1]
```diff
@@ -1,26 +1,95 @@
-from bottle import Bottle, request
+__copyright__ = '''
+Copyright 2017 the original author or authors.
```
Comment (scothis) [inline invoker/http_server.py]:
```suggestion
Copyright 2019 the original author or authors.
```

**Comment by matthewmcnew** [comment]
@dturanski Thanks, for looping me in. I'll give it a look later today.

**Comment by dturanski** [comment]
Merged via https://github.com/projectriff/python3-function-invoker/commit/122c6d0f99c20551b883395f74828d34e762b447

**Pull request closed without merging**